packages/terra-date-time-picker/src/DateTimePicker.jsx

```diff
@@ -83,6 +83,10 @@ const propTypes = {
    * The first parameter is the event. The second parameter is the selected input value in ISO format.
    */
   onSelect: PropTypes.func,
+  /**
+   * Whether an input field for seconds should be shown or not
```

Comment:
Based on some discussion last week, I think this description need to have more information. 

It had sounded like when `showSecond` was used, a date-time format with `hh:mm:ss` time needed to be provided to be valid. Is that still the case?

Reply:
That is still the case.

Reply:
If someone adds the `showSeconds`  but don't update their intial date-time value doesn't include seconds, will this render with the pre-defined value?

Reply:
If seconds are not included in the value, date time picker will assume 00 whether or not `showSeconds` is true. If `showSeconds` is true then the 00 will be rendered in the seconds input box.